src/Fixtures/index.js

```diff
@@ -279,12 +279,12 @@ export default {
       steps: [],
     },
   ],
-  training: [
+  trainings: [
     {
```

Comment:
I guess a training could contain the ids of drills that are included. That would be closer to what we would do with a relational database under the hood ?

Reply:
That is what I tried with the 'drills' field of these training, is that what you mean or something else ?

Reply:
I read too fast, I meant exactly what you did 😄